examples/docker/build-image.sh

```diff
@@ -1,65 +1,65 @@
-#!/bin/sh
```

Comment:
All the line endings in this file have changed to windows style `CR` `LF`, which makes it fail on Linux. You can revert this from the base of the repo by doing `git checkout main -- examples/docker/build-image.sh`

Reply:
Change the end of line sequence as asked.